sdk/src/jobs/tests/JobStatusNotifierTests.cpp

```diff
@@ -82,17 +82,17 @@ TEST_CASE("Job Status Notifier")
    job2->StatusMessage = "Resources.";
 
    system::DateTime sd1, sd2, sd3, sd4;
-   REQUIRE_FALSE(system::DateTime::fromString("2020-03-16T15:43:21.123456", sd1));
-   REQUIRE_FALSE(system::DateTime::fromString("2020-03-13T21:10:11.002244", sd3));
-   REQUIRE_FALSE(system::DateTime::fromString("2020-03-18T05:38:19.997755", sd4));
+   REQUIRE_FALSE(system::DateTime::fromString("2020-03-16T15:43:21.123456",sd1));
+   REQUIRE_FALSE(system::DateTime::fromString("2020-03-13T21:10:11.002244",sd3));
+   REQUIRE_FALSE(system::DateTime::fromString("2020-03-18T05:38:19.997755",sd4));
    job1->SubmissionTime = sd1;
    job2->SubmissionTime = sd2;
    job3->SubmissionTime = sd3;
    job4->SubmissionTime = sd4;
 
    system::DateTime ld1, ld3, ld4;
-   REQUIRE_FALSE(system::DateTime::fromString("2020-03-20T09:57:26.030409", ld1));
-   REQUIRE_FALSE(system::DateTime::fromString("2020-03-14T15:07:08.665544", ld3));
+   REQUIRE_FALSE(system::DateTime::fromString("2020-03-20T09:57:26.030409",ld1));
```

Comment:
Here and elsewhere, if we can fixup the whitespace then we will cut down on the number of changed files which helps readability.